schema.graphql

```diff
@@ -27,12 +27,16 @@ type Token @entity {
 type Order @entity {
   "Trade's OrderUid to hex string"
   id: ID!
```

Comment:
Probably nice to add as a comment for all IDs how do you compose an ID of that entity. i.e. `orderId | txHashString | eventIndex`

Reply:
already done